### PR TITLE
feat(eslint): array includes is acceptable

### PR DESCRIPTION
**Pull request by stevensacks**
This is a questionable rule. As stated in the comment above the rule, Array.includes() is a consistent api with String.includes(). Further, any alleged performance increase is almost assuredly imperceptible in most cases and is thus a premature optimization.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -116,6 +116,11 @@ module.exports = {
     // regular expressions.
     'unicorn/no-unsafe-regex': 'error',
 
+    // Array.includes() is consistent with String.includes()
+    // Whatever minor performance increase this might have is a premature optimization, at best
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-set-has.md
```
Comment (PixnBits) [inline index.js:121]:
```suggestion
    // the rule's performance claim is unsubstantiated and appears to be premature optimization
    // https://github.com/sindresorhus/eslint-plugin-unicorn/issues/495 and 604
```

Comment (stevensacks) [reply index.js]: Done

Comment (stevensacks) [reply index.js]: Done

**Comment by JamesSingleton** [comment]
@stevensacks, just need to resolve conflicts.

**Comment by stevensacks** [comment]
@JamesSingleton I believe I resolved them. Travis successfully built.

**Pull request merged**